bundles/org.openhab.core.io.http.auth/src/main/resources/messages_fr.properties

```diff
@@ -1,26 +1,5 @@
-auth.login.prompt = Connectez-vous pour accorder l'acc�s <b>%s</b> � <b>%s</b>:
-auth.login.fail = Veuillez r�essayer.
-auth.createaccount.prompt = Cr�ez un premier compte administrateur pour continuer.
 
-auth.changepassword.success = Mot de passe modifi�.
 
-auth.createapitoken.prompt = Cr�ez des jetons d'API pour autoriser des services externes.
-auth.createapitoken.name.unique.fail = Un jeton avec le m�me nom existe d�j�, veuillez r�essayer.
-auth.createapitoken.name.format.fail = Nom de jeton invalide, merci d'utiliser uniquement des caract�res alphanum�riques.
-auth.createapitoken.success = Nouveau jeton cr�� :
-auth.createapitoken.success.footer = Veuillez le copier maintenant, il ne sera plus possible de le voir � nouveau.
 
-auth.password.confirm.fail = Les mots de passe ne correspondent pas, veuillez r�essayer.
 
-auth.placeholder.username = Utilisateur
-auth.placeholder.password = Mot de passe
-auth.placeholder.newpassword = Nouveau mot de passe
-auth.placeholder.repeatpassword = Confirmer le nouveau mot de passe
-auth.placeholder.tokenname = Nom du jeton
-auth.placeholder.tokenscope = Port�e (scope) du jeton, facultatif
 
-auth.button.signin = Connexion
-auth.button.createaccount = Cr�er
-auth.button.changepassword = Changer le mot de passe
-auth.button.createapitoken = Cr�er le jeton
-auth.button.return = Retour
```

Comment:
Do you know why some translations are removed in this PR @ghys?
E.g. French should be fully translated: https://crowdin.com/project/openhab-core/fr#
Maybe someone needs to press a button to sync the content?

Reply:
They are translated but not proofread. Crowdin removes them until someone approves them.

Reply:
Ah! That explains. :slightly_smiling_face: So if we don't want any translations that haven't been proofread (which seems to be the current configuration), I think we should just merge this PR. It may even trigger someone to proofread them. :wink: WDYT?

Reply:
I personally do not like to add that much empty files. I checked the "Skip untranslated files" option on Crowdin configuration and triggered a manual sync. Afterwards feel free to merge this.

Reply:
@wborn One more thing. I granted Manager rights to you on openhab-core project in Crowdin.

Reply:
Beware of the encoding of these files - Crowdin can only output UTF-8 and the Properties files should be ISO-8859-1. So I manually converted the encoding from these translated files and submitted a couple of separate PRs back in December. The French strings for the auth pages were never approved on Crowdin and I ignored them because they were made manually along with the code where they're used.

Reply:
We might want to look into https://github.com/openhab/openhab-core/issues/1012#issuecomment-526743703 again:

> In Java 9 and newer, the default encoding for .properties files is UTF-8, and if an invalid UTF-8 byte sequence is encountered it falls back to ISO-8859-1.

Reply:
That would be nice if we no longer need to manually convert the files. It doesn't seem to have removed the empty files though. I will see what happens when I remove the branch and press the button myself now I can do that. :wink:

Reply:
It seems to have worked, see the new https://github.com/openhab/openhab-core/pull/2293. :slightly_smiling_face:

Reply:
From my pov UTF-8 encoded properties files are working now (see #2295).